source/wp-content/themes/wporg-developer-blog/patterns/table-of-contents.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Title: Table of Contents
+ * Slug: wporg-developer-blog/table-of-contents
```

Comment:
I may be mistaken, but I don't see this included anywhere.

Reply:
Yeah, this will be manually inserted by post authors/editors following the opening paragraph of any post that needs a TOC. We just wanted a pattern to ensure consistency. This will be at the discretion of the author. Here's [an example](https://developer.wordpress.org/news/2023/12/10/whats-new-for-developers-december-2023/) on the current site.